testing/himem_test/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config TESTING_HIMEM_TEST
```

Comment:
i guess this should somehow depends on esp32.

Reply:
The configuration option needs to be enabled: CONFIG_ESP32_SPIRAM_BANKSWITCH_ENABLE. And has been added dependency on.